source/_includes/code_examples/code_example.liquid

```diff
@@ -1,24 +1,32 @@
-<div class="code-example ui-tabs{% if code.canSplit %} can-split{% endif %}"{% if code.splitLocation %} style="--split-location: {{ code.splitLocation }}%"{% endif %}>
-  <ul class="ui-tabs-nav ui-helper-reset ui-helper-clearfix">
+<div>
+  {%- if code.scss.size == 1 -%}
```

Comment:
In my mind it might be clearer to keep the `size == 1` logic encapsulated in `generatePlaygroundLinks` (which seems to be doing this same check), and instead check for `{%- if playgroundLinks.scss -%}` here.